.github/pull_request_template.md

```diff
@@ -1,15 +1,67 @@
+<!--
+    Please review the contributing guide before submitting: https://developers.home-assistant.io/docs/android/submit
+    Please, complete the following sections to help the processing and review of your changes.
+    Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
 
-<!-- Thank you for submitting a Pull Request and helping to improve Home Assistant. Please complete the following sections to help the processing and review of your changes. Please do not delete anything from this template. -->
+    Thank you for submitting a Pull Request and helping to improve Home Assistant. You are amazing!
+-->
 
 ## Summary
-<!-- Provide a brief summary of the changes you have made and most importantly what they aim to achieve -->
+<!--
+    Provide a brief summary of the changes you have made and most importantly what they aim to achieve.
+    Don't forget any links that could be useful to the reader. (Github issues, PRs, documentation, articles, ...)
+
+    * What was the motivation behind this change?
+    * What is the impact of the changes on the application?
+-->
+
+## Checklist
+<!--
+    Put an `x` in the boxes that apply. You can also fill these out after
+    creating the PR. If you're unsure about any of them, don't hesitate to ask.
+    We're here to help! This is simply a reminder of what we are going to look
+    for before merging your code.
+-->
+
+- [ ] New or updated tests have been added to cover the changes following the testing [guidelines](https://developers.home-assistant.io/docs/android/testing/introduction).
+- [ ] The code follows the project's [code style](https://developers.home-assistant.io/docs/android/codestyle) and [best_practices].(https://developers.home-assistant.io/docs/android/best_practices).
```

Comment:
The second link is broken

Reply:
All the links are broken until the doc PR is merged.

Reply:
Good catch 👍🏻